src/visualizations/config/adapters/dhis_dhis/index.js

```diff
@@ -34,5 +34,13 @@ export default function ({ store, layout, extraOptions }) {
         config.subText = indicatorType?.displayName
     }
 
+    // DHIS2-10496: show icon on the side of the value if an icon is assigned in Maintenance app and the option is set in DV options
+    const showIcon = Boolean(layout.icons?.length)
+    const icon = metaData.items[metaData.dimensions.dx[0]].style?.icon
```

Comment:
FWIW, this will crash if there's no `dx[0]` item or no match in `items[ ]`...

Reply:
This part has been moved to the DV plugin component.